src/content/add-to-app/ios/project-setup.md

```diff
@@ -5,75 +5,71 @@ description: Learn how to integrate a Flutter module into your existing iOS proj
-For an example using SwiftUI, see the iOS directory in [News Feed App][].
+application as embedded frameworks.
+To embed Flutter in your existing application,
+consider one of the following three methods.
+
+**Use the CocoaPods dependency manager and installed Flutter SDK.**
+: Each time you build the app,
+  it compiles the `flutter_module` from source.
+  (Recommended.)
+
+**Create frameworks for each component.**
+: Create frameworks for the Flutter engine, your compiled Dart code,
+  and all Flutter plugins in Xcode.
+  Embed these frameworks and update your existing application's build settings.
+  This method works for teams that don't want to require every developer
+  to install the Flutter SDK and Cocoapods on their local machines.
+
+**Create frameworks for Dart code and Flutter plugins. Use CocoaPods for the Flutter engine.**
+: Embed the frameworks for your iOS app and the plugins in Xcode,
+  but distribute the Flutter engine as a CocoaPods podspec.
+  This works like the second method,
+  but it provides an alternative to distributing the large Flutter.xcframework.
+
+For examples using an app built with UIKit,
+see the iOS directories in the [add_to_app code samples][].
+For an example using SwiftUI, consult the iOS directory in [News Feed App][].
+
+## Watch the video
+
+To learn more, watch this short video on adding Flutter to an iOS app:
+
+<iframe class="full-width" src="{{site.yt.embed}}/IIcrfrTshTs" title="Learn about how to add Flutter to an existing iOS app" {{site.yt.set}}></iframe>
 
 ## System requirements
 
 Your development environment must meet the
-[macOS system requirements for Flutter][]
-with [Xcode installed][].
-Flutter supports iOS 12 and later.
-Additionally, you will need [CocoaPods][]
-version 1.10 or later.
-
+[macOS system requirements for Flutter][] with [Xcode installed][].
+Flutter supports iOS {{site.appmin.xcode}} and later and [CocoaPods][] {{site.appmin.cocoapods}} or later.
 
 ## Create a Flutter module
 
-To embed Flutter into your existing application, 
-using any of the methods mentioned above, 
-first create a Flutter module.
-
-From the command line, run:
+To embed Flutter into your existing application with any method,
+create a Flutter module first.
+Use the following command to create a Flutter module.
 
 ```console
-cd some/path/
-flutter create --template module my_flutter
+$ cd /path/to
+$ flutter create --template module my_flutter
 ```
 
-A Flutter module project is created at `some/path/my_flutter/`.
-If you are using the first method mentioned above, 
-the module should be created in the same parent directory 
-as your existing iOS app. 
-
+Flutter creates module project under `/path/to/my_flutter/`.
+If you use Method A, the module should be created
+in the same parent directory as your existing iOS app.
 
 From the Flutter module directory, you can run the same `flutter`
 commands you would in any other Flutter project,
 like `flutter run --debug` or `flutter build ios`.
```

Comment:
Should we drop the `--debug` option? It's not clear to me what value it adds

```suggestion
like `flutter run` or `flutter build ios`.
```

Reply:
Re-opening this as the current text still has `--debug`. Is there a reason to keep the `--debug` option here? I don't feel strongly but I do think it's a little bit better :)